acceptance-tests/src/test/java/tech/pegasys/ethsigner/tests/dsl/node/PantheonNode.java

```diff
@@ -56,6 +56,7 @@
   private final Contracts contracts;
   private final Transactions transactions;
   private final Web3j jsonRpc;
+  private final Eth eth;
```

Comment:
Why does this need to be a field variable, as it looks to be only used in the constructor?

Reply:
leftover ... > REMOVED!